apps/preview/package.json

```diff
@@ -14,20 +14,29 @@
   "dependencies": {
     "@custom/schema": "workspace:*",
     "@custom/ui": "workspace:*",
+    "cookie-parser": "^1.4.6",
     "express": "^4.19.2",
+    "express-basic-auth": "^1.2.1",
```

Comment:
flagging as this seems to no longer receive updates since 3 years, and there were mentions of a `v2` for breaking changes but no branch for that exists on the repo

Reply:
Thanks, `1.2.1` still seems to be the latest version. Shouldn't be too much of an issue though, especially because we are only using it as a fallback and favour OAuth / set it by default.